content/design-guidelines/images.mdx

```diff
@@ -0,0 +1,66 @@
+---
+title: Image use
+---
+
+When using a screenshot, it’s best to match the background. We suggest to take a screenshot on a similar color background or use a PNG with a transparent background as this will look the most seamless.
+
+## Usage guidelines:
+
+<DoDontContainer>
+  <Do>
+    <img
+      alt="image-white"
+      src="https://user-images.githubusercontent.com/10384315/54647439-b32e1100-4a5f-11e9-8d85-a4a2e64319ca.png"
+    />
+    <Caption>Use images with a similar or transparent background</Caption>
+  </Do>
+  <Dont>
+    <img
+      alt="image-black"
+      src="https://user-images.githubusercontent.com/10384315/54647548-1f107980-4a60-11e9-9640-8f1721609e44.png"
+    />
+    <Caption>Don't use images with opposite backgrounds to the slide background</Caption>
+  </Dont>
+</DoDontContainer>
+
+### 📍Tip: Seamless image backgrounds
+
+If you find yourself needing to use a specific image but the slide background doesn't match, you can change the background color to match the image's background color.
+
+![bg-change](https://user-images.githubusercontent.com/10384315/56434794-05469a00-628b-11e9-8447-93bb44facbec.gif)
+
+## Adding images
+
+Images can be easily added to any presentation by simply drag and dropping, copy and pasting, or inserting from the toolbar in Keynote, Google Slides, and Figma.
```

Comment:
Let's try to never say: easy, simple, just 😄 

I find "drag and dropping" weird, but leaving it up to you, and my 1 second research [result](https://english.stackexchange.com/questions/92755/drag-dropping-or-dragging-dropping).

```suggestion
Images can be added to any presentation by drag and dropping, copy and pasting, or inserting from the toolbar in Keynote, Google Slides, and Figma.
```